CMakeLists.txt

```diff
@@ -81,15 +81,20 @@ if(TSAN)
 endif()
 
 if(UBSAN)
+  cmake_minimum_required(VERSION 3.13)
```

Comment:
Why is this required?

Reply:
The proper cmake commands for setting linker flags are only available starting from that version.
The patch uses `add_link_options`, which adds a set of flags to any targets created henceforth in the current CmakeLists and its children.